sashimi/main.py

```diff
@@ -8,21 +8,24 @@
 
 
 @click.command()
-@click.option(
-    "--sample-rate",
-    default=40000,
-    help="Sample rate of the board that controls the scanning",
-)
-@click.option("--scanning", default="mock", help="The scanning interface")
-def main(sample_rate, scanning, **kwargs):
-    cli_edit_config("scanning", scanning)
+@click.option("--scanless", is_flag=True, help="Scanless mode")
```

Comment:
so the scanless option is setting scanning to `none` instead of `ni` (see the `if` in the State code below). We should consider unifying config and command line options, instead of this ad-hoc system, e.g. https://pypi.org/project/ConfigArgParse/ (then there is no Click, but I don't think that Click adds much in our case anyway, there is also https://fastcore.fast.ai/script.html)

Reply:
This is something that needs to be debated for a further issue/PR. I agree that the current system feels clumsy. Another option, given the ever-extending number of options in `config.py`, is a text editor. Anyway we have to build one for the instructions so we could put an end these to these two issues at one fell swoop.

However this will be slightly out of the scope of this PR

Reply:
I agree, in a next PR, but in this one I just wanted to point out that to make it work, you have to change the appropriate conf value.